crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -19,6 +19,9 @@ use diesel::{ExpressionMethods, QueryDsl, SelectableHelper};
 use sui_indexer::models_v2::epoch::QueryableEpochInfo;
 use sui_indexer::schema_v2::epochs;
 
+/// Operation of the Sui network is temporally partitioned into non-overlapping,
+/// fixed-duration epochs.
+/// During a particular epoch, the set of validators participating in the network is fixed.
```

Comment:
This feels like info that would be better suited on an epoch's field, unless this is something we ought to know baout epochs?